mlflow/langchain/chat_agent_langgraph.py

```diff
@@ -155,7 +158,147 @@ def _parse_input(
 @experimental
 class LangGraphChatAgent(ChatAgent):
     """
-    Helper class to wrap LangGraph agents as a ChatAgent.
+    Helper class to wrap LangGraph agents as a ChatAgent. Use this class with
+    :py:class:`ChatAgentState` and :py:class:`ChatAgentToolNode`.
+
+    **Open Source LangGraph ChatAgent Example**
```

Comment:
```suggestion
    **LangGraph ChatAgent Example**
```
Also could we mention the langgraph version that this example works? Just in case it breaks in a future release